index.html

```diff
@@ -301,15 +325,19 @@ <h2>Setoriëntatie</h2>
   <!-- Sectie 5.4 -->
   <section id='basisprincipes-referentiedata'>
     <h2>Referentiedata</h2>
-    <p>Referentiadata komt in informatiemodellen doorgaans voor als een enumeratie of codelijst, maar soms ook in de vorm van attributen of complexe datatypes. Referentiedata is een goede indicator van linkbare data. Want het is een manier om verschillende datasets aan elkaar te koppelen of een bepaalde categorisering aan te brengen over verschillende datasets. Ofwel, referentiedata = link naar andere data.</p>
+    <!-- 1 zin uitleg wat we met referentiedata bedoelen toevoegen -->
+    <p>Referentiedata komt in informatiemodellen doorgaans voor als een enumeratie of codelijst, maar soms ook in de vorm van attributen of complexe datatypes. Referentiedata is een goede indicator van linkbare data. Want het is een manier om verschillende datasets aan elkaar te koppelen of een bepaalde categorisering aan te brengen over verschillende datasets. Ofwel, referentiedata = link naar andere data.</p>

```

Comment:
Wat ik zelf ook altijd wel belangrijk vind: iemands referentiedata is de data voor een ander. Dus zeker in een context van linked data geloof ik zelf niet in referentiedata in de klassieke vorm van enumeraties en codelijstjes.

Reply:
opmerking hierover toegevoegd